### DISCUSSION TITLE
git clone 이후 생길 수 있는 빌드 문제

### DISCUSSION THREAD

**Discussion by sunshiningsoo** [ku-ring/app-ios-v2#23] [iOS 개발]
[bbe455f](https://github.com/ku-ring/ios-app/commit/bbe455f0f17744d57f1a6ea2688c5fda872ebd4d) 기준으로 작성된 글입니다.
-----

### 생길 수 있는 빌드 문제
초기에 clone을 받아 프로젝트 빌드를 하게 된다면 하단 스크린샷에 해당하는 문제를 확인 할 수 있습니다.

<img width="500" alt="빌드문제1" src="https://github.com/ku-ring/ios-app/assets/72736657/e2d86006-67dd-4ae9-a493-35a96b0363e4">

위 문제는 패키지 파일의 Tests/Feature/FeedbackFeatureTests 경로에 FeedbackFeatureTests 디렉토리 및 파일이 없어서 문제가 생기는 것인데요.

### 해결 방법
이를 해결하기 위해 `Package.swift`파일의 FeedbackFeatureTests 가 적혀 있는 라인을 주석처리해주세요.

<img width="500" alt="스크린샷 2023-09-26 11 50 37" src="https://github.com/ku-ring/ios-app/assets/72736657/2591f133-d3c6-426f-88ec-349d62d8b791">

### 빌드

1. 프로젝트 `Package cache clean` (File->Packages->Reset Package Caches)을 해주시고,
2. 빌드 Scheme를 `KuringModulePackage` 혹은 `KuringModulePackage-Package` 로 선택 후 빌드
3. 다시 `KuringApp` Scheme 선택 후 빌드

의 과정을 거치면 정상으로 프로젝트 빌드와 시뮬레이터로 결과를 확인해 볼 수 있습니다.

**Comment by x-0o0** [comment]
PR https://github.com/ku-ring/ios-app/issues/25 에서 수정

**Comment by lgvv** [comment]
또 하나의 빌드 문제가 있는데요, swift 5.9가 맞춰지지 않아서 발생하는 에러도 있었습니다.
에러 더블클릭해서 trust and build 해주면 됩니다!

**Reply by x-0o0** [reply]
@lgvv 아 이건 매크로 문제입니다. 매크로 사용시에 발생하는 건데 package 에서 안뜨도록 하는 방법을 찾아볼게요
